blueprints/component/files/__root__/__templatepath__/__templatename__.gts

```diff
@@ -0,0 +1,3 @@
+<template>
+  {{yield}}
+</template>
```

Comment:
I think we can TO types already: https://typed-ember.gitbook.io/glint/environments/ember/template-only-components